core/themes/seven/css/style.css

```diff
@@ -703,6 +706,9 @@ fieldset .fieldset-wrapper {
   border-top-color: #000;
   border-bottom: 0;
 }
+.js[dir="rtl"] fieldset.collapsible .fieldset-legend a:before {
+  right: 0;
```

Comment:
To make this symmetrical, seems as though this should be `right: .6em;`. Also when specifying the RTL version, the left value should be reset, such as `left: auto;`.

Reply:
fixed.